tasks/main.yml

```diff
@@ -1,21 +1,26 @@
 ---
+- import_tasks: cleanup.yml
+  when: systemd_networkd_cleanup | bool
 
-- import_tasks: profiles.yml
+- import_tasks: config.yml
 
 - name: enable systemd-networkd
+  become: true
   service:
     name: systemd-networkd
     enabled: yes
   when: systemd_networkd_network or systemd_networkd_link or systemd_networkd_netdev
 
 - name: start and enable systemd-resolved
+  become: true
   service:
     name: systemd-resolved
     enabled: yes
     state: started
   when: systemd_networkd_enable_resolved
```

Comment:
I assume that if a user disables that after the first run - it will always be enabled and started, but I guess it's more of a bug inherited from forking than something that should be solved with this PR.

Reply:
That's true. I haven't enabled it in kayobe yet, we're still using /etc/resolv.conf. We'd need to support configuring systemd-resolvd to use it.